### PR TITLE
Webrtc client side

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
This is the client side for WebRTC (EE)

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-3412
https://mattermost.atlassian.net/browse/PLT-3943

#### Depends on:
https://github.com/mattermost/platform/pull/3706

#### Checklist
- [ ] Added or updated unit tests (required for all new features)
- [ ] Added API documentation (required for all new APIs)
- [x] All new/modified APIs include changes to the drivers
- [x] Has enterprise changes (https://github.com/mattermost/enterprise/pull/49)
- [x] Has UI changes
- [x] Includes text changes and localization files updated
- [ ] Touches critical sections of the codebase (auth, upgrade, etc.)

This is a re-submit for #3711 but against master

### CODE REVIEW

**Comment by enahum** [comment]
@crspeller build is fixed.

File: webapp/components/webrtc/components/webrtc_header.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2016 ZBox, Spa. All Rights Reserved.
```
Comment (jwilander) [inline webapp/components/webrtc/components/webrtc_header.jsx]: This should be `Copyright (c) 2016 Mattermost, Inc.`, correct?

File: webapp/components/webrtc/components/webrtc_sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2015 Mattermost, Inc. All Rights Reserved.
```
Comment (jwilander) [inline webapp/components/webrtc/components/webrtc_sidebar.jsx]: Should be 2016 :P

File: webapp/stores/webrtc_store.jsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
+/**
+ * Created by enahum on 3/24/16.
+ */
```
Comment (jwilander) [inline webapp/stores/webrtc_store.jsx]: Missing copyright

**Comment by enahum** [comment]
@jwilander done

**Comment by coreyhulen** [comment]
+1, @enahum feel free to merge.

**Pull request merged**